packages/lesswrong/server/emailComponents/EmailInactiveUserSummary.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { defineStyles, useStyles } from "@/components/hooks/useStyles";
+import { postGetPageUrl } from "@/lib/collections/posts/helpers";
+import { getSiteUrl } from "@/lib/vulcan-lib/utils";
+import { truncate } from "@/lib/editor/ellipsize";
+import { EmailUsername } from "./EmailUsername";
+
+export type BestReaction = {name: string, count: number};
+
+const styles = defineStyles("EmailInactiveUserSummary", (theme: ThemeType) => ({
+  root: {
+    background: theme.palette.text.alwaysWhite,
+    color: theme.palette.text.alwaysBlack,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 12,
+    fontWeight: 400,
+    width: 522,
+    maxWidth: "100%",
+    margin: "24px auto",
+  },
+  spacing: {
+    marginBottom: 48,
+  },
+  heading: {
+    fontSize: 16,
+    fontWeight: 700,
+  },
+  mostCommentedPost: {
+    textDecoration: "underline",
+    color: "inherit",
+  },
+  notifications: {
+    fontSize: 11,
+    fontStyle: "italic",
+    textDecoration: "underline",
+    color: "inherit",
+  },
+  post: {
+    marginBottom: 40,
+  },
+  postTitle: {
+    fontSize: 30,
+    fontWeight: 700,
+    lineHeight: "36px",
+    marginBottom: 0,
+    color: "inherit",
+  },
+  postAuthors: {
+    fontWeight: 500,
+    marginBottom: 16,
+  },
+  postBody: {
+    fontFamily: theme.palette.fonts.serifStack,
+    fontSize: 12,
+    fontWeight: 400,
+  },
+  hr: {
+    border: "none",
+    borderBottom: `1px solid ${theme.palette.text.alwaysBlack}`,
+  },
+}));
+
+export const EmailInactiveUserSummary = ({
+  user,
+  karmaChange,
+  bestReaction,
+  mostCommentedPost,
+  unreadNotifications,
+  recommendedPosts,
+}: {
+  user: DbUser,
+  karmaChange: number,
+  bestReaction?: BestReaction,
+  mostCommentedPost?: {post: DbPost, commentCount: number},
+  unreadNotifications: number,
+  recommendedPosts: PostsList[],
+}) => {
+  const classes = useStyles(styles);
+  return (
+    <div className={classes.root}>
+      <p>
+        Hi <EmailUsername user={user} />,
+      </p>
+      <p>
+        We noticed you haven’t been on the EA Forum in a while. Here are some
+        things you’ve missed.
+      </p>
+      <div className={classes.spacing} />
+      <h2 className={classes.heading}>
+        Your notifications
+      </h2>
+      {karmaChange > 0 &&
```

Comment:
<img width="322" height="142" alt="Screenshot 2025-07-21 at 11 39 24" src="https://github.com/user-attachments/assets/a35b25d4-1f5f-48ba-95ae-7048b3297a5a" />
I think this shouldn't appear in the case where the person has no notifications